eng/format-verifier.ps1

```diff
@@ -0,0 +1,77 @@
+[CmdletBinding(PositionalBinding = $false)]
+Param(
+    [string]$logPath,
+    [string]$testPath
+)
+
+$repos = @(
+    "https://github.com/dotnet/format",
+    "https://github.com/dotnet/roslyn"
+    "https://github.com/dotnet/cli",
+    "https://github.com/dotnet/project-system",
+    "https://github.com/Microsoft/msbuild",
+    "https://github.com/dotnet/corefx",
+    #"https://github.com/aspnet/AspNetCore",
```

Comment:
Why not AspNetCore?

Reply:
It alone take more time than the uncommented repos combined.